as-modules/embedded/build.xml

```diff
@@ -63,7 +63,7 @@
             <maven-resource group="org.infinispan" artifact="infinispan-cachestore-jpa" />
         </module-def>
 
-        <module-def name="org.infinispan.cdi" slot="${infinispan.slot}">
+        <module-def name="org.infinispan.cdi.embedded" slot="${infinispan.slot}">
```

Comment:
I think at the very least we should have an org.infinispan.cdi module that acts as an alias for embedded